lib/puppet/pops/validation/checker4_0.rb

```diff
@@ -559,6 +559,8 @@ def internal_check_top_construct_in_module(prog)
     return if namespace_for_file(file) == NO_NAMESPACE
 
     body = prog.body
+    return if(prog.body.is_a?(Model::Nop)) #Ignore empty or comment-only files
+
```

Comment:
:+1: 

Nit, could you remove the parenthesis around the condition:

```ruby
if prog.body.is_a(Model::Nop)
```

Reply:
Done!